meta-ostro-xt/recipes-swupd/swupd-client/swupd-client/efi-combo-trigger.service.new

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Updates the EFI combo application
+BindsTo=update-triggers.target
+
+[Install]
+WantedBy=update-triggers.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/efi_combo_updater
```

Comment:
I wasn't expecting such eagerness in the review :-)
The patch is not final, but I wanted to start checking if something fails in CI.